### PR TITLE
data: Viveksati5143

### PR DESCRIPTION
**Pull request by Viveksati5143**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8005"></a>

### CODE REVIEW

**Comment by Viveksati5143** [comment]
@SaraJaoude @Sindhuinti Please have a look...

**Review by sandramsc — APPROVED** [review]
File looks good Vivek!
welcome_to_linkfree

**Comment by sandramsc** [comment]
> @SaraJaoude @Sindhuinti Please have a look...

FYI: no need to tag the maintainers directly as they get automatically notified already. Any additional tagging creates unnecessary "noise".

File: data/Viveksati5143.json [hunk 1]
```diff
@@ -1,16 +1,61 @@
...
+      "url": "https://github.com/Viveksati5143"
+    },
+    {
+      "icon": "SiLinkedin",
+      "url": "https://www.linkedin.com/in/vivek-sati"
+    }
+  ],
+  "testimonials": [
+    
+  ],
```
Comment (Alkaison) [inline data/Viveksati5143.json:25]: You can remove this section because its empty.

**Review by Alkaison — CHANGES_REQUESTED** [review]

File: data/Viveksati5143.json (hunk 1 above)
Comment (Viveksati5143) [reply data/Viveksati5143.json:25]: Then how will someone add testimonials?

**Review by Alkaison — APPROVED** [review]
All good, the testimonials are added via the issue template for them. 

You can check them here: https://github.com/EddieHubCommunity/LinkFree/issues/new?assignees=&labels=testimonial&projects=&template=testimonial.yml&title=New+Testimonial

**Comment by danishhansari** [comment]
Awesome, Thanks for contribution

**Review by SaraJaoude — APPROVED** [review]
Looks good. 

If you would like to add more sections or remove anything as suggested in the future, you can do so either via a new PR with JSON or through our new Forms.  Here are the Editing Guides for each:

Editing with JSON: https://linkfree.io/docs/how-to-guides/editing-json

Editing with Forms: https://linkfree.io/docs/how-to-guides/editing-forms

**Comment by Viveksati5143** [comment]
> Looks good.
> 
> If you would like to add more sections or remove anything as suggested in the future, you can do so either via a new PR with JSON or through our new Forms. Here are the Editing Guides for each:
> 
> Editing with JSON: https://linkfree.io/docs/how-to-guides/editing-json
> 
> Editing with Forms: https://linkfree.io/docs/how-to-guides/editing-forms

Sure, I'll check it out as I do have to add Events section to my profile.

**Pull request merged**